extension-manifest-v3/src/store/stats.js

```diff
@@ -59,15 +59,9 @@ const Stats = {
   categories: ({ trackers }) => trackers.map((t) => t.category),
   [store.connect]: {
     async get() {
-      const currentTab =
-        chrome.tabs &&
-        (await chrome.tabs.query({ active: true, currentWindow: true }))[0];
-
-      const tabId =
-        currentTab?.id ||
-        (await chrome.runtime.sendMessage({
-          action: 'getCurrentTabId',
-        }));
+      const tabId = await chrome.runtime.sendMessage({
```

Comment:
The `getCurrentTabId` was here only for the fallback for the Firefox iframe limitation. Is using the `chrome.tabs` API not preferred where possible?